e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1

```diff
@@ -653,24 +653,24 @@ az iot dps enrollment create `
 
 if ($EnableIotHubSecuritySolution)
 {
-    Write-Host "`nCreating a self-signed certificate and placing it in $ResourceGroup."
+    Write-Host "`nCreating a self-signed certificate lor LA and placing it in $ResourceGroup."
     az ad app credential reset --id $logAnalyticsAppId --create-cert --keyvault $keyVaultName --cert $ResourceGroup --output none
     Write-Host "`nSuccessfully created a self signed certificate for your application $logAnalyticsAppRegnName in $keyVaultName key vault with cert name $ResourceGroup."
-}
 
-Write-Host "`nFetching the certificate binary."
```

Comment:
This stuff should have been inside the new conditional on line 654. The console writes don't make it evident, and using a secret name of the rg for the LA app name is an odd choice. I didn't realize it was all part of the same logic, but now I see it is.